assets/js/popup.js

```diff
@@ -0,0 +1,67 @@
+const popup = document.getElementById('myPopup');
+const span = document.getElementsByClassName('close')[0];
+
+const arr2 = [
+  'Data Dashboard Health Care',
+  'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry standard',
+  'HTML',
+  'BootStrap',
+  'Ruby',
+  'See Live',
+  'See Source',
+  'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry Ipsum is simply dummy text of the printing and typesetting industry.',
+];
```

Comment:
- [ ] Kindly be noticed that the JavaScript code implements an array of objects to store JavaScript objects containing the information for each project. Each object contains at least the following information about the project: name, description, featured image, technologies, link to the live version, and link to the project source code. Ie:
```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}]
```